packages/issuer/package.json

```diff
@@ -8,6 +8,9 @@
   "scripts": {
     "build": "tsc"
   },
+  "dependencies": {
+    "@sphereon/openid4vci-common": "workspace:../common"
```

Comment:
Any reason not to use workspace:* ?

Reply:
I was getting errors that the dependency couldn't not be resolved and after changing it to `workspace:../common` it started to work. But I'm not sure anymore whether it is the solution because me and @sksadjad faced some weird behavior all day long.